apps/api/src/services/delegation.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { InjectRedis } from '@liaoliaots/nestjs-redis';
+import Redis from 'ioredis';
```

Comment:
Hmm, I wonder if we need redis in this service? Doesn't seem like we are using it directly.

Reply:
removed ✅